fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/BitmapValueIndexMaintainer.java

```diff
@@ -279,6 +279,11 @@ public CompletableFuture<Tuple> evaluateAggregateFunction(@Nonnull IndexAggregat
                 .thenApply(combined -> Tuple.from(combined.asByteArray()));
     }
 
+    @Override
+    public CompletableFuture<Void> mergeIndex() {
```

Comment:
Since the contract of this is likely to vary from index type to index type and require additional state, I think index operations (which already exist) are probably better suited for doing Lucene merges.

Reply:
We could possibly defer this to another pull request.